torch/optim/adadelta.py

```diff
@@ -5,9 +5,33 @@
 
 
 class Adadelta(Optimizer):
-    """Implements Adadelta algorithm.
-
-    It has been proposed in `ADADELTA: An Adaptive Learning Rate Method`__.
+    r"""Implements Adadelta algorithm.
+
+    .. math::
+       \begin{aligned}
+            &\rule{110mm}{0.4pt}                                                                 \\
+            &\textbf{input}      : \gamma \text{ (lr)}, \: \theta_0 \text{ (params)},
+                \: f(\theta) \text{ (objective)}, \: \rho \text{ (decay)},
+                \: \lambda \text{ (weight decay)}                                                \\
+            &\textbf{initialize} : square\_avg_0 \leftarrow 0,
```

Comment:
Why not use `v_0` like the other algorithms: https://pytorch.org/docs/master/generated/torch.optim.RMSprop.html#torch.optim.RMSprop ?

Reply:
done !